awx/api/views.py

```diff
@@ -145,6 +145,15 @@ def destroy(self, request, *args, **kwargs):
         # Still allow deletion of new status, because these can be manually created
         if obj.status in ACTIVE_STATES and obj.status != 'new':
             raise PermissionDenied(detail=_("Cannot delete running job resource."))
+        elif not obj.events_processed:
+            # Prohibit deletion if job events are still coming in
+            if obj.finished + dateutil.relativedelta.relativedelta(minutes=1) < now():
```

Comment:
@AlanCoding my brain had some serious trouble parsing this, and I'm not sure why.  Should this be _greater than_?